_i18n/ar/resources/user-guides/change-restore-height.md

```diff
@@ -0,0 +1,63 @@
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+### Table of contents
+
+- [What is the wallet restore height?](#what-is-the-wallet-restore-height)
+- [Why Monero wallets have a restore height?](#why-monero-wallets-have-a-restore-height)
+- [What happens if I set a wrong restore height?](#what-happens-if-i-set-a-wrong-restore-height)
+- [How to change the wallet restore height on Monero GUI](#how-to-change-the-wallet-restore-height-on-monero-gui)
+- [My wallet is showing a wrong balance after changing the restore height](#my-wallet-balance-is-wrong-after-changing-the-restore-height)
+
+### What is the wallet restore height?
+
+The wallet restore height is a setting that tells the wallet software at which block height it should start scanning the @blockchain looking for @transactions, a process that is known as wallet synchronization. Ideally it should be the number of the block which included the first incoming transaction to your wallet.
```

Comment:
```suggestion
The wallet restore height is a setting that tells the wallet software which block it should start scanning the @blockchain from looking for @transactions, a process that is known as wallet synchronization. Ideally it should be the number of the block which included the first incoming transaction to your wallet.
```

I would avoid using "height" if we can. People are often confused by the term and this guide could be very useful in clarifying things.

Reply:
fixed